lib/omni_auth/strategies/forem.rb

```diff
@@ -0,0 +1,36 @@
+require "omniauth-oauth2"
+
+module OmniAuth
+  module Strategies
+    class Forem < OmniAuth::Strategies::OAuth2
+      option :name, :forem
+
+      option :client_options, {
+        site: ApplicationConfig["PASSPORT_OAUTH_URL"] || "https://passport.forem.com".freeze,
+        authorize_url: "/oauth/authorize"
+      }
+
+      uid { raw_info["id"] }
+
+      info do
+        {
+          email: raw_info["email"],
+          name: raw_info["name"],
+          user_nickname: raw_info["username"]
+        }
+      end
+
+      extra do
+        skip_info? ? {} : { raw_info: raw_info }
+      end
+
+      def raw_info
+        @raw_info ||= access_token.get("/api/v0/me").parsed
```

Comment:
If the result of `parsed` is a normal Ruby hash it could make sense to use `.with_indifferent_access` and use symbols above.

```suggestion
        @raw_info ||= access_token.get("/api/v0/me").parsed.with_indifferent_access
```